src/sparsezoo/model/utils.py

```diff
@@ -122,24 +111,57 @@ def load_files_from_stub(
     params = None
     if isinstance(stub, str):
         stub, params = parse_zoo_stub(stub=stub, valid_params=valid_params)
-    _LOGGER.debug(f"load_files_from_stub: loading files from {stub}")
-    response = download_get_request(
-        args=stub,
-        force_token_refresh=force_token_refresh,
+    _LOGGER.debug(f"load_files_from_stub: calling  files from {stub}")
+
+    arguments = get_model_metadata_from_stub(stub)
+
+    api = GraphQLAPI()
+
+    models = api.fetch(
+        operation_body="models",
+        arguments=arguments,
+        fields=["modelId", "modelOnnxSizeCompressedBytes"],
```

Comment:
Related to my comment below, but could you just include "files", "training_results", and "benchmark_results" in this fields list?

Reply:
same as above